compiler_gym/util/gym_type_hints.py

```diff
@@ -2,8 +2,8 @@
 #
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
-from typing import Any, Dict, List, Optional, Tuple, TypeVar, Union
 from enum import Enum
+from typing import Any, Dict, List, Optional, Tuple, TypeVar, Union
```

Comment:
I'd probably move this to another PR since it's an unrelated (but valid) change

Reply:
Yeah good catch, it crept into this PR as a tiny fixup 🙂